### PR TITLE
Forces lowercase before logging queries (closes #136).

### LINKED ISSUE #136: Metrics should be case insensitive
When logging queries, they should be forced to lowercase.

### PR DESCRIPTION
**Pull request by chuckharmston**
r? @6a68

### CODE REVIEW

File: recommendation/mozlog/tests/test_middleware.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
         'frend': ''
     }
 }
-DUMMY = 'Hello, frend.'
+DUMMY = 'hello, frend.'
```
Comment (chuckharmston) [inline recommendation/mozlog/tests/test_middleware.py]: Prevents other tests from breaking.

File: recommendation/mozlog/middleware.py [hunk 2]
```diff
@@ -56,11 +56,12 @@ def request_summary(response):
     log['t'] = request.finish_time - request.start_time
 
     if query:
+        query = query.lower()
         log['predicates.query_length'] = len(query) > 20
         log['predicates.is_protocol'] = (re.match(IS_PROTOCOL, query) is not
-                                          None)
+                                         None)
```
Comment (chuckharmston) [inline recommendation/mozlog/middleware.py]: Drive-by style fix.

**Comment by 6a68** [comment]
R+, looks like it needs a rebase, though

**Comment by chuckharmston** [comment]
I don't know why our rebases have been so brittle. This is one-line change that has been entirely unrelated to anything that has been committed in the meantime.
